cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -216,7 +220,15 @@ protected Future<Void> manualRollingUpdate(Reconciliation reconciliation, KafkaC
                     if (!podNamesToRoll.isEmpty())  {
                         // There are some pods to roll
                         KafkaConnectRoller roller = new KafkaConnectRoller(reconciliation, connect, operationTimeoutMs, podOperations);
-                        return roller.maybeRoll(podNamesToRoll, pod -> RestartReasons.of(RestartReason.MANUAL_ROLLING_UPDATE));
+                        return roller.maybeRoll(podNamesToRoll, pod -> RestartReasons.of(RestartReason.MANUAL_ROLLING_UPDATE))
+                            .recover(error -> {
+                                if (featureGates.continueReconciliationOnManualRollingUpdateFailureEnabled()) {
+                                    LOGGER.warnCr(reconciliation, "Reconciliation will be continued even though manual rolling update failed");
```

Comment:
I think you should log the error which was raised. Maybe something like:

 ```suggestion
                                    LOGGER.warnCr(reconciliation, "Manual rolling update failed (reconciliation will be continued)", error);
```

The same would apply to the other two places.